src/fastgltf.cpp

```diff
@@ -2210,6 +2214,56 @@ std::size_t fg::GltfDataBuffer::getBufferSize() const noexcept {
 }
 #pragma endregion
 
+#pragma region AndroidGltfDataBuffer
+#if defined(__ANDROID__)
+fg::AndroidGltfDataBuffer::AndroidGltfDataBuffer(AAssetManager* assetManagerIn) noexcept : assetManager{assetManagerIn} {}
+
+bool fg::AndroidGltfDataBuffer::loadFromAndroidAsset(const fs::path& path, std::uint64_t byteOffset) noexcept {
+    if (assetManager == nullptr) {
+        return false;
+    }
+    
+    using namespace simdjson;
+
+    const auto filenameString = path.string();
+
+    AAsset* file = AAssetManager_open(assetManager, filenameString.c_str(), AASSET_MODE_BUFFER);
```

Comment:
Optional, but probably better is to (1) use `auto*` here and (2) wrap it with a unique_ptr and a custom deleter:
```cpp
auto assetDeleter = [](AAsset* file) { AAsset_close(file); }
std::unique_ptr<AAsset, decltype(assetDeleter)> file(AAssetManager_open(...), assetDeleter);
```
This guarantees that the file gets closed no matter if anything should throw an exception.